cluster/images/etcd/Makefile

```diff
@@ -97,13 +97,18 @@ else
 
 	# Download etcd in a golang container and cross-compile it statically
 	# For each release create a tmp dir 'etcd_release_tmp_dir' and unpack the release tar there.
+	arch_prefix=""
+        ifeq ($(ARCH),arm)
```

Comment:
nit: this indent looks mis-aligned?

Reply:
Hi @BenTheElder , it's must be "space" instead of "tab" cause "tab" will make interpret it as shell command while `ifeq` is not, make it as this to avoid the failure to run `make` with this `Makefile`.